src/envo/modules/chemical_concentration.csv

```diff
@@ -1,31 +1,88 @@
-defined_class,defined_class_label,chemical,chemical_label,material,material_label
-ENVO:3100001,,CHEBI:15422,Adenosine 5-triphosphate,ENVO:00002006,water
-ENVO:3100002,,CHEBI:28315,alloxanthine,ENVO:00002006,water
-ENVO:3100003,,CHEBI:28425,alpha-carotene,ENVO:00002006,water
-ENVO:3100004,,CHEBI:28938,ammonium,ENVO:00002006,water
-ENVO:3100005,,CHEBI:30033,bacteriochlorophyll a,ENVO:00002006,water
-ENVO:3100006,,CHEBI:17579,beta-carotene,ENVO:00002006,water
-ENVO:3100007,,CHEBI:23042,carotene,ENVO:00002006,water
-ENVO:3100008,,CHEBI:18230,chlorophyll a,ENVO:00002006,water
-ENVO:3100009,,CHEBI:27888,chlorophyll b,ENVO:00002006,water
-ENVO:3100010,,CHEBI:16900,chlorophyllide a,ENVO:00002006,water
-ENVO:3100011,,CHEBI:15379,dioxygen,ENVO:00002006,water
-ENVO:3100012,,CHEBI:73113,divinyl chlorophyll a,ENVO:00002006,water
-ENVO:3100013,,CHEBI:73115,divinyl chlorophyll b,ENVO:00002006,water
-ENVO:3100014,,CHEBI:5186,fucoxanthin,ENVO:00002006,water
-ENVO:3100015,,CHEBI:15996,Guanosine 5-triphosphate,ENVO:00002006,water
-ENVO:3100016,,CHEBI:16240,hydrogen peroxide,ENVO:00002006,water
-ENVO:3100017,,CHEBI:16136,hydrogen sulfide,ENVO:00002006,water
-ENVO:3100019,,CHEBI:28838,lutein,ENVO:00002006,water
-ENVO:3100020,,CHEBI:16183,methane,ENVO:00002006,water
-ENVO:3100021,,CHEBI:25501,neoxanthin,ENVO:00002006,water
-ENVO:3100022,,CHEBI:17632,nitrate,ENVO:00002006,water
-ENVO:3100023,,CHEBI:16301,nitrite,ENVO:00002006,water
-ENVO:3100024,,CHEBI:17045,nitrous oxide,ENVO:00002006,water
-ENVO:3100025,,CHEBI:44882,peridinin,ENVO:00002006,water
-ENVO:3100026,,CHEBI:26020,phosphate,ENVO:00002006,water
-ENVO:3100027,,CHEBI:29241,silicate(4-),ENVO:00002006,water
-ENVO:3100028,,CHEBI:27295,violaxanthin,ENVO:00002006,water
-ENVO:3100029,,CHEBI:27547,zeaxanthin,ENVO:00002006,water
-ENVO:3100034,,CHEBI:26675,silicic acid,ENVO:00002006,water
-ENVO:3100036,,CHEBI:28966,chlorophyll,ENVO:00002006,water
+defined_class,defined_class_label,chemical,chemical_label,material,material_label

```

Comment:
Shouldn't affect the autogen and import, but I'm curious as to why all the lines have been replaced.
@kaiiam is this an Excel artifact? It's better to avoid Excel as it sometimes does mysterious things in the background

Reply:
@pbuttigieg I believe the lines were changed for the reason I had previously mentioned:

> I believe the reason the diff file is so large is that the line encoding changed from pc (which we had set it to previously) to unix which I'm now working on.

I don't believe this is an excel issue but rather a line ending by computer type. As we had previously decided if we all use unix line endings for this file it shouldn't be a problem. 

It is my understading that it is expected that the changes are not yet in the envo-edit.owl file. They will compiled upon release. I believe this is what happens during the make process:

This line will run the patterns file, generating the concomitant .owl files. 

```
modules/%.owl: modules/%.csv patterns/%.yaml curie_map.yaml
```

This line will merge all the module .owl files into the envo-edit.owl
```
# Merge in all module files, whilst removing the corresponding import
envo-edit-module-merge.owl: envo-edit.owl
	$(OWLTOOLS) $(USECAT) $<  $(patsubst %, --remove-import-declaration $(OBO)/envo/modules/%.owl, $(MODS)) -o $@.tmp
	$(OWLTOOLS) $(USECAT) $@.tmp  $(ALL_MODS_OWL) --merge-support-ontologies -o $@
```

Reply:
The line endings explain it.